### PR TITLE
Migrate to Stimulus part 2

### PR DESCRIPTION
**Pull request by andychow326**
Ref #1951

### CODE REVIEW

File: authui/src/resendButton.ts [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
+        this.animHandle = requestAnimationFrame(tick);
+      }
+    };
+
+    this.animHandle = requestAnimationFrame(tick);
+  }
+
+  disconnect() {
+    if (this.animHandle != null) {
+      cancelAnimationFrame(this.animHandle);
```
Comment (louischan-oursky) [inline authui/src/resendButton.ts:42]: We should set `this.animHandle` to `null` after cancelling the animation frame. Generally speaking, a cleanup method may be called more than once. The cleanup method should be able to be run more than once, and running it the second time should have no effect. Here, the same animation handle will be freed twice and it is a problem.

File: authui/src/messageBar.ts [hunk 2]
```diff
@@ -1,46 +1,30 @@
...
-
-    const onCloseButtonClick = (e: Event) => {
-      e.preventDefault();
-      e.stopPropagation();
-      target.classList.add("hidden");
-    };
-
-    // Close the message bar before cache the page.
-    // So that the cached page does not have the message bar shown.
-    // See https://github.com/authgear/authgear-server/issues/1424
```
Comment (louischan-oursky) [inline authui/src/messageBar.ts:26]: Keep this comment

File: authui/src/resendButton.ts (hunk 1 above)
Comment (andychow326) [reply authui/src/resendButton.ts:42]: Done, please take a look

File: authui/src/messageBar.ts (hunk 2 above)
Comment (andychow326) [reply authui/src/messageBar.ts:26]: Done, please take a look

**Pull request merged**